### DISCUSSION TITLE
Change project name

### DISCUSSION THREAD

**Discussion by Machion** [withlogicco/django-prose#116] [Ideas]
Looking for a WYSIWYG editor for Django it confused me very much, that besides django-prose there are proseMirror and django-prose-editor basing on it. 
Maybe it makes more sense to name the project/package for example django-trix, to make it clear it's based on Trix? 
Is there any connection between django-prose and proseMirror?

**Comment by akalipetis** [comment]
Hey, this has already been discussed here: https://github.com/withlogicco/django-prose/issues/115
